pkgs/development/python-modules/scipy/default.nix

```diff
@@ -1,34 +1,41 @@
 {
   lib,
   stdenv,
-  fetchFromGitHub,
-  fetchpatch,
   fetchurl,
   writeText,
-  xcbuild,
   python,
   buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+
+  # build-system
   cython,
   gfortran,
   meson-python,
   nukeReferences,
-  pkg-config,
   pythran,
-  wheel,
+  pkg-config,
   setuptools,
-  hypothesis,
-  pytest7CheckHook,
-  pytest-xdist,
-  numpy,
-  pybind11,
-  pooch,
-  xsimd,
+  xcbuild,
+
+  # buildInputs
   # Upstream has support for using Darwin's Accelerate package. However this
   # requires a Darwin user to work on a nice way to do that via an override.
   # See:
   # https://github.com/scipy/scipy/blob/v1.14.0/scipy/meson.build#L194-L211
   blas,
   lapack,
+  pybind11,
+  pooch,
+  xsimd,
+
+  # dependencies
+  numpy,
+
+  # tests
+  hypothesis,
+  pytest7CheckHook,
+  pytest-xdist,
```

Comment:
I also don't think necessarily the instructions tell us to put `tests` inputs after `dependencies` - putting these were they were and adding a title should be enough IMO.

Reply:
As for the rest of the inputs, they should be listed in the same order as they appear in within the derivation.
As `nativeCheckInputs = ...` is below `dependencies`, then I followed this order.

I agree that this may appear quite annoying but when working on and reviewing PRs, it is very enjoyable to find a consistency and rigor in the code.

Reply:
> I agree that this may appear quite annoying but when working on and reviewing PRs, it is very enjoyable to find a consistency and rigor in the code.

OK, I can live with _this_...